recipe/meta.yaml

```diff
@@ -1,25 +1,24 @@
-{% set version = "2.2.1" %}
+{% set name = "cloudpickle" %}
+{% set version = "3.0.0" %}
 
 package:
-  name: cloudpickle
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: cloudpickle-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/c/cloudpickle/cloudpickle-{{ version }}.tar.gz
-  sha256: d89684b8de9e34a2a43b3460fbca07d09d6e25ce858df4d5a44240403b6178f5
+  url: https://github.com/cloudpipe/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 49cc8581ce305630b1e179bc083069b21b45db4731e5e3f2ea6fc40b131e1c55
 
 build:
   number: 0
-  skip: true # [py<36]
-  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
+  skip: true # [py<38]
+  script:  {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
```

Comment:
I don't think `--ignore-installed --no-cache-dir -vv` is necessary anymore but I suppose it doesn't hurt either.

Reply:
At least one `-v` is needed for us to view what's going on. `-vv` is excessive`.

Reply:
Thanks guys! I'll make a quick little change!